src/asciidoc/docs/develop/useGit.adoc

```diff
@@ -66,37 +66,23 @@ The Koji repositories are stored at
 
 To clone your Koji repository locally, you need these strings:
 
-* The *repository URL* on `\https://projects.koji-cdn.com/`.
 * The *username* that you created when you registered with Koji.
 * An *access token*, which you must generate and store securely.
 
-You need to use the repository URL only once, you should know your username by heart, and the access token is a 32-character, randomly-generated hexadecimal number, which you need to note in a safe place.
-
-=== Obtaining the repository URL
-
-To find the URL of the Koji repository:
-
-. On the left side of the Koji editor, go to *Advanced > Repository Info*.
-+
-The Repository Info tab shows links to two repositories.
-+
-image::remote-repository.png[Remote repository URL in the Koji editor,title="Remote repository URL"]
-. Copy the URL for the remote repository.
-+
-The *remote repository* is the personal repository for your template.
-+
-NOTE: The *upstream repository* is the repository from which your project was forked.
-If your intention is to propose improvements to the original project, then you can modify your remote repository and subsequently create pull requests, so that the developer can evaluate your proposed changes and update the project accordingly.
+You should know your username by heart, and the access token is a 32-character, randomly-generated hexadecimal number, which you need to note in a safe place.
 
 === Obtaining an access key
 
 To clone your Koji repository locally, you must generate an access key and store it securely.
 Access keys are strong, randomly-generated, 32-character hexadecimal strings that you generate to allow programmatic access to your Koji projects.
 
-. From any page on https://withkoji.com[withkoji.com], click your profile picture in the top right, and then click *Developer Portal*.
+. From *Project Details*, click *Clone*.
+A dialog similar to the following will be displayed.
 +
-. Click *My Access Keys*.
-. Click *Generate New Key* in the top right.
+image::git-clone.png[title="File VCC"]
```

Comment:
Remove/replace caption. This is not a File VCC. :)
Also, to avoid the "Figure 1." on the caption, add the `:figure-caption!:` attribute to the page